charms/kfp-api/tox.ini

```diff
@@ -50,10 +50,10 @@ description = Apply coding style standards to code
 commands =
     # uncomment the following line if this charm owns a lib
     # codespell {[vars]lib_path}
-    codespell {toxinidir}/. --skip {toxinidir}/.git --skip {toxinidir}/.tox \
-      --skip {toxinidir}/build --skip {toxinidir}/lib --skip {toxinidir}/venv \
-      --skip {toxinidir}/.mypy_cache \
-      --skip {toxinidir}/icon.svg --skip *.json.tmpl
+    codespell {toxinidir}/. --skip {toxinidir}/./.git --skip {toxinidir}/./.tox \
+      --skip {toxinidir}/./build --skip {toxinidir}/./lib --skip {toxinidir}/./venv \
+      --skip {toxinidir}/./.mypy_cache \
+      --skip {toxinidir}/./icon.svg --skip *.json.tmpl
```

Comment:
Can all of this go into pyproject.toml?

And maybe skip all dot folders? (. /**), if pattern is supported by codespell

Reply:
Do we want this?
What if we want to check `.github/workflow/`?
I will check regarding `./**`, but not sure what to do with `pyproject.toml`